targets/unity-v2/source/Entity/ScreenTimeTracker.cs

```diff
@@ -31,13 +31,14 @@ public class ScreenTimeTracker : IScreenTimeTracker
         private Guid focusId;
         private Guid gameSessionID;
         private bool initialFocus = true;
+        private bool isSending = false;
         private DateTime focusOffDateTime = DateTime.UtcNow;
         private DateTime focusOnDateTime = DateTime.UtcNow;
 
         private Queue<EventContents> eventsRequests = new Queue<EventContents>();
 
         private EntityModels.EntityKey entityInfo = new EntityModels.EntityKey();
-        private const String eventNamespace = "com.playfab.events.";
+        private const String eventNamespace = "com.playfab.events.analytics";
```

Comment:
Please confirm this line with Siva and/or Brent.
The left is what I was asked to use earlier today.
It seems unclear what people actually expect here.

Reply:
fixed